### ISSUE TITLE
[Bug] StoryStoreV7 does not work with .svelte story files

### ISSUE THREAD

**Issue by IanVS** [storybookjs/builder-vite#361]
### What version of `vite` are you using?

2.9.6

### System info and storybook versions

This repo

### Describe the Bug

Reported in https://discord.com/channels/486522875931656193/826188253849190450/969388479303798846, and confirmed by me in our own svelte example, svelte story files do not load when using `storyStoreV7`.  We create the `storiesOf` function correctly, it seems, so I think this is an issue with storybook not processing svelte files.

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by joshwooding** [comment]
Yeah, this is a problem with Storybook https://github.com/storybookjs/storybook/issues/16673
